woocommerce/woocommerce-gateway-stripe
# Add a GitHub Action workflow to run QIT E2E Integration tests.

Fixes #3530 

## Changes proposed in this Pull Request:
This PR adds a GitHub Action workflow to run QIT E2E integration tests for each pull request. The tests are located in the [integration tests repository](https://github.com/woocommerce/woocommerce-gateway-integration-qit), and this action will trigger the test run on every pull request.

## Testing instructions
Make sure that QIT E2E tests GitHub action is running and passing on this PR.

---


**Post merge**
